doc/sphinx/language/core/inductive.rst

```diff
@@ -3,6 +3,10 @@ Inductive types and recursive functions
 
 .. _gallina-inductive-definitions:
 
+The :cmd:`Inductive` command allows to define types by case on the inhabitants of the type. On the contrary of types defined by the command :cmd:`Variant`, constructors can recursively have arguments in the type being defined. So, inductive types include natural numbers, lists, well-founded trees, etc. Inhabitants of inductive type can recursively nest only a finite number of constructors. So, they are well-founded. This distinguishes them from :cmd:`CoInductive` types whose constructors can be nested infinitely. In Coq, :cmd:`Variant` types thus correspond to the common subset of inductive and coinductive types that are non-recursive.
```

Comment:
```suggestion
The :cmd:`Inductive` command allows defining types by cases on the inhabitants of the type. In contrast, types defined by the :cmd:`Variant` command, constructors can recursively have arguments in the type being defined. Inductive types include natural numbers, lists and well-founded trees. Inhabitants of inductive type can recursively nest only a finite number of constructors. So, they are well-founded. This distinguishes them from :cmd:`CoInductive` types, whose constructors can be infinitely nested. In Coq, :cmd:`Variant` types thus correspond to the common subset of inductive and coinductive types that are non-recursive.
```

Please split this into multiple lines that are ~80 characters long.

This needs some work.
"define types by case on the inhabitants of the type" doesn't make sense to me unless you mean "inhabitants of another type"
How about defining "well-founded" and adding maybe add it to the glossary?
Also, would you give a concrete example of a coinductive that's infinitely nested?  That's not at all obvious.

IIRC there is not a clear explanation (with example) of the difference between a coinductive and an inductive.  Could you add that?